### PR TITLE
Add Arduino guide

### PR DESCRIPTION
**Pull request by Zifzaf**
- Add arduino.rst file which explains how to setup the SGP40 with
  an Arduino board.
- Add two images needed for this guide in the image folder.

Check the following:

 - [na] Breaking changes marked in commit message
 - [na] Changelog updated
 - [na] Code style cleaned (ran `make style-fix`)
 - [na] Tested on actual hardware

### CODE REVIEW

File: docs/sgp40_voc_index/arduino.rst [hunk 1]
```diff
@@ -0,0 +1,110 @@
+SGP40 on Arduino Platforms
+==========================
+
+This guide describes how to get the new sgp40 working on arduino
```
Comment (abrauchli) [inline docs/sgp40_voc_index/arduino.rst:4]: It'll likely not stay new ;)

```suggestion
This guide describes how to get the sgp40 gas sensor working on Arduino
```

File: docs/sgp40_voc_index/arduino.rst [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+platforms. It offers a step by step guide to get basic funcionality working.
+
+Preperation
+-----------
+
+First of all note that the most commenly used Arduino boards have a logic level of 5v but the
```
Comment (abrauchli) [inline docs/sgp40_voc_index/arduino.rst:10]: For all instances, Volt uses a capital V symbol. https://en.wikipedia.org/wiki/Volt

File: docs/sgp40_voc_index/arduino.rst [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+SGP40 can only run on 3.3v and lower. We offer two sugsestions how to solve this. First you could
+use a level shifter board to connect your SGP40, alternitivly you could use one of the few Arduino
```
Comment (abrauchli) [inline docs/sgp40_voc_index/arduino.rst:12]:
```suggestion
use a [level shifter](https://en.wikipedia.org/wiki/Level_shifter) to connect your SGP40, alternatively you could use one of the few Arduino 
```

maybe we can provide a link to a shifter board?

File: docs/sgp40_voc_index/arduino.rst [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+This guide describes how to get the new sgp40 working on arduino
+platforms. It offers a step by step guide to get basic funcionality working.
+
+Preperation
+-----------
+
+First of all note that the most commenly used Arduino boards have a logic level of 5v but the
+SGP40 can only run on 3.3v and lower. We offer two sugsestions how to solve this. First you could
+use a level shifter board to connect your SGP40, alternitivly you could use one of the few Arduino 
+boards which run on 3.3v. For this guide we did the later and used the Arduino Nano 33 BLE.
```
Comment (abrauchli) [inline docs/sgp40_voc_index/arduino.rst:13]:
```suggestion
boards which run at 3.3v. For this guide we did the later and used the Arduino Nano 33 BLE.
```

In general I'd use less prose to have more "meat".. maybe

"The SGP40 only runs at Voltages between 1.6V - 3.6V. Solutions include
* Using a level shifter
* Using a board like the Arduino Nano 33 BLE that runs at 3.3V"

File: docs/sgp40_voc_index/arduino.rst [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+Windows: https://www.arduino.cc/en/Guide/Windows 
+Linux: https://www.arduino.cc/en/Guide/Linux
+Mac OS: https://www.arduino.cc/en/Guide/MacOSX
+
+Connecting the Sensor
+---------------------
+
+Your sensor has the four different connectors: VCC, GND, SDA, SCL.
+Now you need to connect the sensor to the correct pins on your board. To find out
+which pins are the correct ones cunsult the pinout of your sepesific board. You
```
Comment (abrauchli) [inline docs/sgp40_voc_index/arduino.rst:26]:
```suggestion
which pins are the correct ones consult the pinout of your specific board. You
```

File: docs/sgp40_voc_index/arduino.rst [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+can find these on the Arduino website. Look for SDA to connect the sensors SDA pin, 
+for SCL to connect the SCL pin, GND for the GND pin and a 3.3v supplie for the VCC pin.
```
Comment (abrauchli) [inline docs/sgp40_voc_index/arduino.rst:28]:
```suggestion
for SCL to connect the SCL pin, GND for the GND pin and a 3.3v supply for the VCC pin. 
```

File: docs/sgp40_voc_index/arduino.rst [hunk 7]
```diff
@@ -0,0 +1,110 @@
...
+Then you can type the name of your board in the search bar and install the upcoming package.
+
+|Boardmanager|
+
+Now you only need to upload the code to your board. Connect your Arduino board with your computer
+using an usb cable. Then you'll need to select the entry in the Tools, Board menu that corresponds 
+to your Arduino board. Finaly you also need to select the right port where your Arduino is conected. 
+For this navigate to Tools, Port and select the one inicating that your board is connected there
+(name of your board at the end in brackets). If no port is indicating that a board is connected 
+it's likely to be COM3. To now upload your code to your board click the arrow icon in the top lefthand corner.
```
Comment (abrauchli) [inline docs/sgp40_voc_index/arduino.rst:55]: ...looks like this line is too long

File: docs/sgp40_voc_index/arduino.rst [hunk 8]
```diff
@@ -0,0 +1,110 @@
...
+-  Check if you connected the sensor correctly, so all cables are fully
+   pluged in and connected to the correct header.
+-  Check if you have enabled I2C on the Raspberry Pi. For this redo the
+   steps on "Enable the I2C interface in the Raspberry Pi config" in the
+   guide above.
+
+VOC Index is zero
+~~~~~~~~~~~~~~~~~
+
+For the first 100 iterations it is always zero, so just wait a little.
```
Comment (abrauchli) [inline docs/sgp40_voc_index/arduino.rst:105]: Explain why, people will be curious to know.

```suggestion
During the initialization phase of the VOC Index algorithm, which is the first 100 iterations, the VOC Index is hardcoded to zero.
```

**Review by abrauchli — CHANGES_REQUESTED** [review]
Thanks for the guide, It's definitely a good start and provides major value as is. Reflecting, my two concerns are
* it reads more like a blog-post than a step-by-step guide. It's not bad, but I think people could benefit from more conciseness if it was slightly less wordy. @rnestler, what do you think?
* Use a spell-checker

File: docs/sgp40_voc_index/arduino.rst (hunk 8 above)
Comment (Zifzaf) [reply docs/sgp40_voc_index/arduino.rst]: I did know either ...

**Review by rnestler — APPROVED** [review]
Fixed some typos and restructured it.

**Review by rnestler — APPROVED** [review]

**Pull request merged**